index.js

```diff
@@ -43,6 +43,72 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Ramda
+//.
+//. Sanctuary started as a complementary project to [Ramda][], aiming to
+//. provide some of Ramda's functionality in a "safer" way, by utilizing
```

Comment:
Hmm. I think grammatically "its" refers to Sanctuary rather than Ramda. From context it may be apparent that we're referring to Ramda, but I'd prefer not to ask readers to make this switch. I'll give this sentence more thought.